run

```diff
@@ -64,6 +64,7 @@ if [[ -z "$DBOX_SKIP_UPDATE" ]]; then
 	fi
 fi
 
+chmod -R a+rx /opt/dropbox/
```

Comment:
This is overly broad, and will set the executable bit on many files that aren't executable.  I used

    chmod a+rx /opt/dropbox/dropbox-lnx*/*.so

Reply:
Thanks!